### DISCUSSION TITLE
[firebase-storage] useStorageEmulator() should set default-bucket

### DISCUSSION THREAD

**Discussion by jarrodonlo** [firebase/flutterfire#7767] [Feature request]
### Calling `useStorageEmulator(host, port)` should also set the `bucket =  "default-bucket"`.

Although Firebase Emulator uses "default-bucket" as bucket name...
(On package `firebase_storage`) calling `useStorageEmulator()` does not use this bucket by default, which seems like it would be expected behavior, as all users must do this manually. Instead, continuing to use bucket setup from `flutterfire` cli generated `FirebaseOptions`.

Instead of doing one line...
```dart
FirebaseStorage.instance.useStorageEmulator('localhost', 9199);
```
the user must currently do two lines to use emulator...
```dart
FirebaseStorage.instance.bucket = 'default-bucket'; // must be done prior! (order matters)
FirebaseStorage.instance.useStorageEmulator('localhost', 9199);
```
this was a tripping point for myself, I feel like it will be for other new users. 
![Firebase-Emulator-Suite](https://user-images.githubusercontent.com/94813690/147981332-8362a2c8-e25f-4c41-8476-0eac8bad0d2b.png)

**Comment by william-kung** [comment]
Thank you for the advice.  I was struggling for hours to connect StorageEmulators.  It works after adding the suggested line. 
 `FirebaseStorage.instance.bucket = 'default-bucket';`
